### PR TITLE
feat: [IDP-1935]: pipeline and executions added in old nav for IDP

### PR DESCRIPTION
**Pull request by bhavya-sinha**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [x] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

File: src/modules/75-idp/components/IDPAdminSideNav/__test__/IDPAdminSideNav.test.tsx [hunk 1]
```diff
@@ -37,6 +37,10 @@ describe('IDP Sidenav', () => {
       </TestWrapper>
     )
     expect(getByText('common.plugins')).toBeVisible()
-    expect(container).toMatchSnapshot()
+    expect(getByText('idp.urlAllowList')).toBeVisible()
```
Comment (chetanvekariya-harness) [inline src/modules/75-idp/components/IDPAdminSideNav/__test__/IDPAdminSideNav.test.tsx:40]: Thanks for removing snapshot. 👏

File: src/modules/10-common/RouteDefinitions.ts [hunk 2]
```diff
@@ -2442,6 +2442,15 @@ const routes = {
   toIDPOAuthConfig: withAccountId(() => '/idp-admin/oauth'),
   toIDPAllowListURL: withAccountId(() => '/idp-admin/allowlist-url'),
   toScorecards: withAccountId(() => '/idp-admin/scorecards'),
+  toIDPProjectSetup: withAccountId(() => `/idp-admin/project-setup`),
```
Comment (chetanvekariya-harness) [inline src/modules/10-common/RouteDefinitions.ts:2445]: Shouldn't we have RouteDefinitions file in 75-idp?

**Review by chetanvekariya-harness — APPROVED** [review]

File: src/modules/10-common/RouteDefinitions.ts (hunk 2 above)
Comment (bhavya-sinha) [reply src/modules/10-common/RouteDefinitions.ts:2445]: There is a common routedefinition file for all the modules

**Review by jenil-harness — APPROVED** [review]
LGTM

**Pull request merged**